pdns/lua-auth4.cc

```diff
@@ -136,6 +136,9 @@ bool AuthLua4::axfrfilter(const ComboAddress& remote, const DNSName& zone, const
 
 
 bool AuthLua4::updatePolicy(const DNSName &qname, QType qtype, const DNSName &zonename, DNSPacket *packet) {
+  // default decision is all goes
```

Comment:
Does this need a documentation update?

Reply:
no, it's just explicitly documented here.